virtio-devices/src/vsock/unix/muxer.rs

```diff
@@ -32,7 +32,7 @@
 //!    `VsockConnection`.
 //!
 //! The muxer gets notified about all of these events, because, as a `VsockEpollListener`
-//! implementor, it gets to register a nested epoll FD into the main VMM epolling loop. All
+//! implementor, it gets to register a nested epoll FD into the main VMM epollin loop. All
```

Comment:
I think this was supposed to be epolling - let's write it as "epoll()ing loop"

Reply:
Thanks. I was really on the fence on this one